### PR TITLE
Use charm/bundle supplied expert data.

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Use guiproxy to staging and dd cmars/mattermost, the values in the top right box should be as supplied by the charmstore

Note: the supplied content at the time of writing is providing extra header content, so it will not match 'design'. As this is free-form markdown the layout is up to the entity author.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8655/

File: jujugui/static/gui/src/app/components/deployment-flow/expert-intro/expert-intro.js [hunk 1]
```diff
@@ -105,34 +106,14 @@ class DeploymentExpertIntro extends React.Component {
                 title="Juju expert partners">
                 <div className="deployment-expert-intro__plan-details">
                   <h3>
-                    From $3577 per month
+                    {price}
                   </h3>
                   <div className="deployment-expert-intro__plan-description">
                     Default plan with standard support
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/deployment-flow/expert-intro/expert-intro.js]: Should this still be here? Looks like it should be removed...

Comment (hatched) [reply jujugui/static/gui/src/app/components/deployment-flow/expert-intro/expert-intro.js]: Nope this stays as the price that is supplied by the expert here is using the default plan with standard support. Modifications to that price are further down in the flow.

Comment (huwshimi) [reply jujugui/static/gui/src/app/components/deployment-flow/expert-intro/expert-intro.js]: Oh I see!

**Comment by huwshimi** [comment]
Thanks @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**